src/CommitmentsV2/SFA.DAS.CommitmentsV2.Types/PaymentStatus.cs

```diff
@@ -4,6 +4,7 @@ namespace SFA.DAS.CommitmentsV2.Types
 {
     public enum PaymentStatus : short
     {
+        PendingApproval = 0,
```

Comment:
don't think this is a valid payment status. Might be a valid ApprenticeshipStatus?

Reply:
removed